src/styles/index.ts

```diff
@@ -3359,7 +3370,8 @@ const styles = (theme: ThemeColors) =>
         },
 
         cardSectionTitle: {
-            lineHeight: variables.lineHeightXXLarge,
+            fontSize: variables.fontSizeLarge,
```

Comment:
I assume that these style changes are already approved by design?

Reply:
Yes, it is taken from Figma

Reply:
ok just wanna make sure that this is intentional by design.

this branch:
<img width="711" alt="Screenshot 2024-02-05 at 6 45 55 PM" src="https://github.com/Expensify/App/assets/108292595/f78c65e0-e1f8-49bc-acf9-80c9ffd1707d">


staging:
<img width="711" alt="Screenshot 2024-02-05 at 6 46 00 PM" src="https://github.com/Expensify/App/assets/108292595/e3092647-c38c-454e-9c8f-3283882ee9c6">

Reply:
[Design link](https://www.figma.com/file/Xv3yHJooZcxRGkXbAcTZD5/%23ideal-nav-(Ideal-Nav)?type=design&node-id=3125-183280&mode=design&t=PT7ZlFNu5IEimO94-4) - I think it is ok